src/python/tests/unittests/test_controller/test_exclude_patterns.py

```
@@ -0,0 +1,88 @@
+# Copyright 2017, Inderpreet Singh, All rights reserved.
+
+import unittest
+
+from system import SystemFile
+from controller import filter_excluded_files
+
+
+class TestFilterExcludedFiles(unittest.TestCase):
+    def test_empty_pattern_returns_all(self):
+        files = [SystemFile("a.txt", 10, False), SystemFile("b.nfo", 20, False)]
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)